### DISCUSSION TITLE
nwg-iso 2023.10.19

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#204] [Changelog]
**Changes to nwg-live ISO**

- all the installation, for all user accounts, performed by one script;
- all AUR packages provided in the local repository;
- the text editor & web browser selection menu abandoned, in favour of Mousepad and Firefox.

**Comment by aadityapageni** [comment]
Awesome

**Reply by nwg-piotr** [reply]
Thank you! Which version do you use, by the way? I'm going to cancel nwg-mini, as maintaining two different ISOs takes too much time.
